.github/workflows/parametric-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Parametric Tests
```

Comment:
Could you please rename it into something clarifying what it does?
I think these are APM-only System Tests, right?

Reply:
Sure, thanks for pointing out!
Yes these are only APM-related tests currently but afaik there is no reason preventing other products from writing parametric tests.
The main difference compared to system-tests is that parametric tests only cover the library (not the agent) and it supports testing w/ multiple config options.
Happy to rename it `APM Parametric Tests` if that makes sense, wdyt? LMK if you think of a better name!

Reply:
SGTM, thanks :-)